atomic_defi_design/Dex/NewUpdateModal.qml

```diff
@@ -19,6 +21,17 @@ Dex.MultipageModal
         return 3
     }
 
+    Component.onCompleted:
+    {
+        if (Dex.API.app.updateCheckerService.updateInfo.status === "recommended" ||
+            Dex.API.app.updateCheckerService.updateInfo.status === "required" )
+        {
+            root.open()
+        }
```

Comment:
Can we add else case to add to logs if status is ok or api returns status_code != 200?
```
        else
        {
            console.log(Dex.API.app.updateCheckerService.updateInfo.status)
        }
```

Reply:
Yes ofc

Reply:
I tried this locally and seems would need to add to logs on back end not front.